lib/attr.gd

```diff
@@ -37,8 +37,14 @@
 ##
 ##  <Description>
 ##  This info class (together with <Ref InfoClass="InfoWarning"/> is used
-##  for messages about attribute storing  being  disabled  (at  level  2)  or
-##  enabled (level 3). It may be  used  in  the  future  for  other  messages
+##  for messages about attributes. The current warnings are:
+##  <List>
+##    <Item> Attribute storing  being  disabled  (level  2) </Item>
+##    <Item> Atribute storing being enabled (level 3)</Item>
+##    <Item> Trying to assign a non-mutable attribute a different value (level 3)</Item>
```

Comment:
At least in German, an enumeration should either consist of complete sentences (starting in upper case, ending with a period/question mark/exclamation mark), or an enumeration as part of one big sentence (then starting in lower case, and usually ending with a comma, semicolon, or possibly a conjoining word like "and", "or). In the latter case it should also fit together with the start of the sentence before the list. Not sure how much of that applies in English, but here's a suggestion for this line (I'd also make suggestions for the first two lines, but I don't understand them at all, so I cannot do that :/).

So if we were to start the sentence with "The current warnings are shown under the following circumstances:" as I suggest above, we could have this list item read like this:

```suggestion
##    <Item>when trying to assign to non-mutable attribute which already is set to a different value (level 3);</Item>
```

Of course we could also do it completely differently. It's also not really the style or grammar I am concerned about, but rather comprehensibility of the text.